spec/store_model/types/one_spec.rb

```diff
@@ -91,6 +91,23 @@
         include_examples "for unknown attributes"
       end
 
+      context "when saving model" do
+        let(:custom_product_class) do
+          build_custom_product_class do
+            attribute :configuration, Configuration.to_type
+          end
+        end
+
+        let(:subject) do
+          product = custom_product_class.create(
```

Comment:
let's extract it to the separate `let` statement

Reply:
so I extracted the product and subject in my latest commit. was It something like that you had in mind?